### DISCUSSION TITLE
Не прошла транзакция на биржу Bitmart

### DISCUSSION THREAD

**Discussion by sia1968** [Chia-Network/chia-blockchain#17577] [Support]
В чиа кошельке есть запись
Подтверждено на блоке  4951936. Монеты списались. https://xchscan.com/ru/blocks/4951936 показывает хеш заголовка 0x8afb6efc82d88231ee4628b15448399618db614a06ffe3a73bc8b11460702dbf Тех. поддержка Bitmart пишет:

 I am unable to to search your TXID for 0x8afb6efc82d88231ee4628b15448399618db614a06ffe3a73bc8b11460702dbf
 
Much appreciated if you could send us a explorer link instead for us to investigate further. 
подскажите что делать?

**Comment by BrandtH22** [comment]
Hey @sia1968 , as a heads up I am using translation software. The hash you provided is for the block itself and not for a transaction. Can you provide the xch address to which the funds were sent?

The exchange will likely want the coin ID of the coin that was sent to the xch address

---
Привет, в качестве предупреждения я использую программное обеспечение для перевода. Хэш, который вы указали, предназначен для самого блока, а не для транзакции. Можете ли вы указать xch-адрес, на который были отправлены средства? Биржа, скорее всего, захочет получить идентификатор монеты, которая была отправлена на адрес xch

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
